client/blocks/jitm/index.jsx

```diff
@@ -36,6 +36,7 @@ export const JITM = ( {
 			onDismiss={ onDismiss( currentSite.ID, id, featureClass ) }
 			event={ `jitm_nudge_click_${ id }` }
 			href={ `https://jetpack.com/redirect/?source=jitm-${ id }&site=${ currentSite.domain }` }
+			target={ '_blank' }
```

Comment:
This is fine. But why not literal `target="_blank"`

Reply:
no reason other than it looks better ;) beside all the others, to me, anyway.